client/js/lounge.js

```diff
@@ -437,7 +437,7 @@ $(function() {
 	var settings = $("#settings");
 	var options = $.extend({
 		desktopNotifications: false,
-		colors: false,
+		coloredNicks: true,
```

Comment:
Just for the record:

> 16:07	**+xPaw**	astorije, looks OK except for the renamed settings, i'll test it today or tomorrow
> 16:08	**+astorije**	xPaw, what's wrong with the renamed settings§?
> 16:08	**+xPaw**	backwards compat
> 16:08	**+xPaw**	I would rather leave renaming settings to Max-P's PR
> 16:08	**+astorije**	I know it breaks settings, but if we are jumping 2.0.0 I'm not too complexed about it
> 16:09	**+astorije**	To be honest, I am not too happy having a class that is system specific like that simply called "colors"
> 16:09	**+astorije**	And leaving the renaming to Max-P's PR means changing 32 * 4 lines
> 16:09	**+xPaw**	i specifically mean the setting name
> 16:09	**+xPaw**	class can stay
> 16:09	**+astorije**	Oh right, my bad
> 16:09	**+astorije**	Yeah
> 16:10	**+astorije**	I'd say the earlier the better, especially if we jump 2.x
> 16:10	**+astorije**	If we end up not doing it (which I doubt more and more), then fine, but otherwise...
> 16:10	**+xPaw**	eh, fuck it, fine
> 16:10	**+xPaw**	i'll test it later to make sure nothing broke
> 16:10	**+astorije**	lol
> 16:10	**+astorije**	Sure
> 16:10	**+astorije**	I tested the 4 themes in private browsing fyi